step1/service.js

```diff
@@ -0,0 +1,89 @@
+import { state } from "./state.js";
+import { baseTemplate } from "./template.js";
+
+
+export const addItem = function (event) {
+    event.preventDefault();
+    const $app = document.querySelector('#app');
+    const $appenderForm = $app.querySelector('form[name="appenderForm"]');
+    const content = $appenderForm.querySelector('input').value.trim();
+    if (content.length===0){
+        return alert("아이템 내용을 입력해주세요.");
+    }
```

Comment:
```suggestion
    if (content.length === 0) {
        return alert("아이템 내용을 입력해주세요.");
    }
```
사소하지만 이렇게 띄어쓰기만 잘 해줘도 가독성이 좋아진답니다!